portal/patient/_machine_config.php

```diff
@@ -28,7 +28,11 @@
     GlobalConfig::$PORTAL = 1;
     require_once(__DIR__ . "/../../interface/globals.php");
 } else {
-    OpenEMR\Common\Session\SessionUtil::portalSessionCookieDestroy();
+    try {
+        OpenEMR\Common\Session\SessionUtil::portalSessionCookieDestroy();
```

Comment:
odd that an exception would happen here. is there a use case that causes this? (i suspect issue is instead a bug that would fix)

Reply:
@bradymiller 
Below is why plus globals that just dies with Missing Site Id. This can be caused if site times out in patient documents under certain circumstances. Really session manager shouldn't throw exception in first place IMO.
```

[29-Aug-2023 11:00:33 America/New_York] PHP Warning:  session_destroy(): Trying to destroy uninitialized session in C:\xampp\htdocs\openemr-master\src\Common\Session\SessionUtil.php on line 268
[29-Aug-2023 11:00:33 America/New_York] PHP Stack trace:
[29-Aug-2023 11:00:33 America/New_York] PHP   1. {main}() C:\xampp\htdocs\openemr-master\portal\patient\index.php:0
[29-Aug-2023 11:00:33 America/New_York] PHP   2. require_once() C:\xampp\htdocs\openemr-master\portal\patient\index.php:14
[29-Aug-2023 11:00:33 America/New_York] PHP   3. require_once() C:\xampp\htdocs\openemr-master\portal\patient\_machine_config.php:34
[29-Aug-2023 11:00:33 America/New_York] PHP   4. require_once() C:\xampp\htdocs\openemr-master\interface\globals.php:623
[29-Aug-2023 11:00:33 America/New_York] PHP   5. authCloseSession() C:\xampp\htdocs\openemr-master\library\auth.inc.php:100
[29-Aug-2023 11:00:33 America/New_York] PHP   6. OpenEMR\Common\Session\SessionUtil::coreSessionDestroy() C:\xampp\htdocs\openemr-master\library\auth.inc.php:128
[29-Aug-2023 11:00:33 America/New_York] PHP   7. OpenEMR\Common\Session\SessionUtil::standardSessionCookieDestroy() C:\xampp\htdocs\openemr-master\src\Common\Session\SessionUtil.php:150
[29-Aug-2023 11:00:33 America/New_York] PHP   8. session_destroy() C:\xampp\htdocs\openemr-master\src\Common\Session\SessionUtil.php:268
```